tests/test_validation.py

```diff
@@ -523,7 +523,7 @@ def test_brief_response_essential_requirements():
 
 
 def test_brief_response_nice_to_have_requirements():
-    schema_name = "brief-responses-digital-outcomes-and-specialists-digital-specialists"
+    schema_name = "brief-responses-digital-outcomes-and-specialists-2-digital-specialists"
```

Comment:
If we previously had tests for both essentials and nice to have's for DOS1, should we not have tests for both for DOS2?